crates/uv-python/src/managed.rs

```diff
@@ -252,9 +252,9 @@ impl ManagedPythonInstallation {
     /// The path to this toolchain's Python executable.
     pub fn executable(&self) -> PathBuf {
         if cfg!(windows) {
-            self.path.join("install").join("python.exe")
```

Comment:
Do we need to keep compatibility with the previously installed toolchain that has a `install` subdirectory?

Reply:
I think we might as well retain compatibility there, yeah. Can we just check if `install` is a dir?